modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You installed the OpenShift CLI (`oc`).
+* Your oc client must be updated to at least verion 4.13.0. 
+* Your {product-title} cluster must be installed on either the AWS or Azure platform. 
++
+For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_.
+
+
+.Procedure
+. Verify `RetrievedUpdates=True` in Cluster Version Operator (CVO) conditions by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade
+----
++ 
+For more information about cluster version condition types, see _Understanding cluster version condition types_.
+
+. Set the channel to `stable-<4.y>` or `fast-<4.y>` with the following command: 
++
+[source,terminal]
+----
+$ oc adm upgrade channel <channel>
+----
++
+For more information about channels, see _Understanding update channels and releases_.
+
+. Migrate to the mult-architecture payload with following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-multi-arch
+----
+
+.Verification 
+
+* After you execute the upgrade command, your cluster now uses the multi-arch payload, but the cluster operators continue to upgrade in the background. You can monitor the upgrade process by running the following command: 
++
+[source,terminal]
+----
+$ watch  'oc get co; oc get mcp; oc get clusterversion'
```

Comment:
I think we can replace `oc get clusterversion` with `oc adm upgrade`. And it would be great to mention what the complete status looks like. For `oc get co`, we expect AVAILABLE=true, PROGRESSING=false, DEGRADED=false on all COs. For `oc get mcp`, we expect UPDATED=true, UPDATING=false, DEGRADED=false on all pools. For `oc adm upgrade`, we expect not seeing `info: An upgrade is in progress`

Reply:
happy to apply these changes. Do you have an example response I can include with this info? :) and what should we advise if the response is not what we're expecting?

Reply:
`Progressing=False` is a bit fiddly, depending on what else the cluster is up to.  E.g. here is a cluster with `network` flipping in and out of  `Progressing=True` as it autoscales compute nodes, even though it is not moving between different release images during this time window.

![image](https://user-images.githubusercontent.com/209920/230972115-625e6db4-0e82-4f3d-bd5d-e3976b559931.png)

Reply:
Ideally `oc adm upgrade`, `oc get clusteroperators`, and `oc get machineconfigpools` should all include enough context to give the next step in trying to resolve any issues.  For some output, like `oc adm upgrade`'s:

```
Working towards ${VERSION}: 106 of 841 done (12% complete), waiting on etcd, kube-apiserver
```

no action is necessary.  For other output, such as:

```
Cluster operator kube-storage-version-migrator is not available
```

the implicit ask (which the cluster-version operator should probably adjust to make explicit) is that the cluster admin go look at the `kube-storage-version-migrator` ClusterOperator's conditions for that operator's suggestions about what's going wrong and what might be done about it.  The current limited ClusterVersion `message` is backstopped by [the existing `ClusterOperatorDown` alert][1], which is trying to call in the cluster admin to take a look, and is pointing the admin at the ClusterOperator user interfaces.

[1]: https://github.com/openshift/cluster-version-operator/blob/cce97c25292b263a5a549b2bb8ce82ed7f2d4923/install/0000_90_cluster-version-operator_02_servicemonitor.yaml#L96-L104